### ISSUE TITLE
Multiple forks not providing all cards when custom json is made (XDG, TAD, TSIT so far)

### ISSUE THREAD

**Issue by Qwinn1** [joaquimguimaraes/farmr#78]
DogeChia, a fork for which a blockchain.json is not currently provided, allows farming of XDG.  I created my own xdg.json (feel free to add it as a template to future versions if you like, and I'd be honored if that contribution would be acknowledged :):

```
{
    "Binary Name": "dogechia",
    "Folder Name": ".dogechia",
    "Currency Symbol": "XDG",
    "Minor Currency Symbol": "mojo",
    "Net": "mainnet",
    
    "Block Rewards": 2.0,
    "Blocks Per 10 Minutes": 32.0,
    
    "Online Config": true,
    
    "Ports": {
        "harvester": 6770,
        "farmer": 16759,
        "fullNode": 6769,
        "wallet": 16761,
        "daemon": 56669
      }
}
```

It's mostly working great (and I was thrilled to see that it picked up the correct display of my cold wallet balance same as any other blockchain), but there are two cards I like having up that are oddly missing:  Skipped Blocks and Complete Sub Slots.  I only started farming this last night so maybe it requires more than 24 hours activity to generate these?  Just figured I'd ask.

**Comment by Qwinn1** [comment]
Huh.  For a minute there, I thought it was my mistake for failing to put in a "Folder Name" (I've added it to the above json cause, why not), but it doesn't fix the issue.  Putting in optional config and log paths doesn't fix it either.

**Comment by Qwinn1** [comment]
Final bit of oddness to report:  If I choose the menu option to view the xdg report, it looks great:

```
*Full Node Stats*
64 complete Sub Slots
18.5% orphan Signage Points
Lost sync 2 times, skipped 27 blocks
15:06 from block 148818 to 148845
15:06 from block 148818 to 148845
Synced to block height 149018
Local blockchain version 1.0.9

```
All that information seems to be correct, so it's not that it isn't finding it.

**Comment by joaquimguimaraes** [comment]
Sync losses are logged twice in chia, it is a known chia bug.
Have skipped blocks and complete sub slots reappeared?

**Comment by joaquimguimaraes** [comment]
Regarding xdg.json, can you make a github fork of farmr, add that file to blockchain/xdg.json and make a pull request? This way your name can be added to the list of contributors :)

**Comment by Qwinn1** [comment]
The cards have not appeared yet under XDG, no.  I tried restarting and refreshing everything too.

And cool, I'll do the whole fork/pull-request thing soon as I work out how to again, it's been like 3 years since last I did that.

**Comment by Qwinn1** [comment]
Figured I'd add - something that seems unusual about XDG is that the main directory under which it is installed does not fit the normal "-blockchain" naming convention.  The main directory you install to is just "doge-chia".  Dunno if that would be relevant, but it's the only thing I'm seeing unusual about it out of the 4 other forks I've been working with lately.

**Comment by joaquimguimaraes** [comment]
hmmm and are the log files located in that folder?

**Comment by Qwinn1** [comment]
No.  The log and config directories are under /home/user/.dogechia/mainnet, same as all the others.

**Comment by joaquimguimaraes** [comment]
most of the parameters are optional, same with the Ports, they only enhance your experience and unlock extra features 
so, no, not using Folder Name doesn't break anything

**Comment by Qwinn1** [comment]
Okay, it gets worse.  Looks like my XDG attempt was actually my best attempt so far, at least I'm getting SOME log information.

I tried adding Tad (TAD) and Silicoin (tSIT) by creating .jsons... and they show up, but they're missing even more cards.  Longest Response Time and Missing Challenges are missing as well, from every harvester report.  The logs do exist, are set to INFO correctly and look normal.  The farmr reports when selected aren't showing that information though.  Let me know if you want me to post the .jsons I made and tried out.

Is setting "Minor Currency Symbol": "mojo" correccly in the .json's important for the parser's purposes?  That's the only setting in the .json that I'm not sure about.  Don't know how I'd go about determining their equivalents to a mojo.

**Comment by joaquimguimaraes** [comment]
One of the reasons these forks might have broken log parsing is if they're still keeping chia name in the logs.
I'm working on a new update which should replace most log parsing features with RPC calls, and this should make it run smoother for forks.

**Comment by joaquimguimaraes** [comment]
And yes, Minor Currency Symbol is important when getting hot wallet balances

**Comment by Qwinn1** [comment]
To first post:  Awesome!  Really looking forward to that update, thank you!

To second post:  Ah, okay, I'm only using cold wallets and their balances appear to be working perfectly fine on every fork, so I'll presume it doesn't matter much to me at least.

**Comment by Qwinn1** [comment]
Just checked, btw:  Silicoin has plenty of references to chia in the log, but can't find any chia references in the Tad log, yet they're both missing the same set of cards.  Weird.

**Comment by Qwinn1** [comment]
Pull request submitted per your request.

**Comment by joaquimguimaraes** [comment]
This was fixed in v1.7.1
